optuna/integration/chainermn.py

```diff
@@ -142,3 +157,28 @@ def _check_chainermn_availability():
             'ChainerMN can be installed by executing `$ pip install chainermn`. '
             'For further information, please refer to the installation guide of ChainerMN. '
             '(The actual import error is as follows: ' + str(_import_error) + ')')
+
+
+class ChainerMNPruner(BasePruner):
```

Comment:
Can we make this class private?

Reply:
I made `ChainerMNPruner` and `ChainerMNObjectiveFunc` private (https://github.com/pfnet/optuna/pull/286/commits/97df5bbb5106d8ef666c32aa40ee1fe129cc0423). Thank you for your advice.